eng/scripts/Language-Settings.ps1

```diff
@@ -520,6 +521,45 @@ function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName)
   return $packages
 }
 
+# function is used to filter packages to submit to API view tool
+# Function pointer name: FindArtifactForApiReviewFnV2
+function Find-java-Artifacts-For-ApireviewV2($artifactDir, $packageInfo)
```

Comment:
This function are almost identical and we can just pass an optional parameter at the caller that has -packageInfo and -pkgName to keep it compatible with the other languages and we don't need this duplication. We can clean the others up one-by-one if we choose.